2-mandatory/3-stocks.js

```diff
@@ -34,7 +34,21 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+    // for storing averages of each company's stock over 5 days
+    let averagePriceOverFiveDays = [ ];
+
+    // This loop cycles through each stock, the i is the 5 day stock for AAPL, MSFT etc.
+    for (i = 0; i < closingPricesForAllStocks.length; i++) {
+        let totalStock = 0;
+        closingPricesForAllStocks[i].forEach((element) => {
```

Comment:
Suggestion: Instead of doing `* 100` on the calculated average at the end of the function do a `* 100` on each element when adding it to totalStock. This lets you do more of the math with integers which reduces the floating point inaccuracies caused by math with floating point numbers.